molecule/enable_libpodstats/verify.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Copyright 2021 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- import_playbook: ../common/verify.yml
+
+- name: "Verify collectd exec plugin"
+  hosts: collectd-test
+  tasks:
+    - name: Check that the libpodstats conf file exists
+      stat:
+        path: '/etc/collectd.d/libpodstats.conf'
+      register: output
+      failed_when: not output.stat.exists
+
+    - name: "Check that the TypesDB has been updated"
+      slurp:
+        src: /etc/collectd.conf
+      register: output
+      changed_when: false
+      failed_when:
+        - not "TypesDB \"/etc/collectd.d/types.db.libpodstats\"" in output.content | b64decode
+
```

Comment:
The only thing I might add here is a check on the contents of the config file, namely
```xml
LoadPlugin "libpodstats"
<Plugin "libpodstats">
</Plugin> 
```
As of now, that is the only configuration that can exist.